netmiko/cisco/cisco_s300.py

```diff
@@ -21,4 +21,9 @@ def session_preparation(self):
         self.set_terminal_width(command='terminal width 511')
         # Clear the read buffer
         time.sleep(.3 * self.global_delay_factor)
-        self.clear_buffer()
+
+    def save_config(self):
+        """ Saves config """
+        self.enable()
+        self.send_command(command_string='write', expect_string='[Yes')
```

Comment:
hi, is this tested in physical switch? in my cisco s300 switch, the expect_string is (Y/N)
switch797d1f#write
Overwrite file [startup-config].... (Y/N)[N] ?N

Reply:
Yes, I have an updated branch for this work and I had previously changed that to execute a 'write memory' and to confirm with a 'Y').

```
 25     def save_config(self, cmd='write memory', confirm=True, confirm_response='Y'):
 26         return super(CiscoS300SSH, self).save_config(cmd=cmd, confirm=confirm)
```